vow/promise.go

```diff
@@ -39,12 +30,16 @@ func newPromise(name string, args ...string) *promise {
 }
 
 func (p *promise) Run(w io.Writer) (err error) {
-	buf := new(bufCloser)
-	p.cmd.Stdout = buf
-	p.cmd.Stderr = buf
+	var buf bytes.Buffer
```

Comment:
Although we're never closing the buffer manually, I believe the exec package is. 

If I remember correctly, this was here so that when we kill the command the output doesn't print out.